packages/camllib/camllib.1.3.3/opam

```diff
@@ -2,28 +2,29 @@ opam-version: "2.0"
 maintainer: "Nicolas Berthier <m@nberth.space>"
 authors: ["Bertrand Jeannet"]
 homepage: "https://www.inrialpes.fr/pop-art/people/bjeannet/bjeannet-forge/camllib/index.html"
-bug-reports: "https://gforge.inria.fr/projects/bjeannet/"
-# SVN repositories not supported (yet).
-# dev-repo: "svn://scm.gforge.inria.fr/svnroot/bjeannet/pkg/camllib/branches/opam-packaging"
+bug-reports: "https://framagit.org/nberth/camllib/-/issues"
+dev-repo: "git+https://framagit.org/nberth/camllib.git"
 license: "LGPL-2.1-only"
 build: [
-  ["sh" "./configure"]
+  ["sh" "./configure"
+        "--disable-profiling" {ocaml >= "4.08"}]
   [make]
 ]
 install: [
   [make "install"]
 ]
 depends: [
-  "ocaml" {>= "4.02.0" & < "4.08"}
+  "ocaml" {>= "4.02.0" & < "5"}
   "ocamlfind" {build}
   "ocamlbuild" {build}
   "stdlib-shims"
 ]
+available: [ os != "win32" & os != "cygwin" ]
 synopsis: "Utility Library (including various datatypes)"
 url {
-  src: "http://nberth.space/pool/camllib/camllib-1.3.3.tar.gz"
+  src: "https://nberth.space/pool/camllib/camllib-1.3.3.tar.gz"
```

Comment:
The original tarball can be downloaded from https://gitlab.ocamlpro.com/OCamlPro/opam-repository/-/blob/cached/cache/md5/59/59424473c5a79c4a38da034de36b0293
would it be possible to keep the old hashes and upload this one on `nberth.space` instead?

Reply:
Awesome, thanks a lot!